roles/update/templates/l3_agent_stop_ping.sh.j2

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+#
+# Script which stops the ICMP connectivity check on validates that
+# there is no packet loss.
+set -eu -o pipefail
+
+# In seconds.
+LOSS_THRESHOLD=${1:-{{ loss_threshold|default(60)|int }}}
+LOSS_THRESHOLD_PERCENT=${2:-{{ loss_threshold_percent|default(1)|int }}}
```

Comment:
wondering does loss_threshold and/or loss_threshold_percent belongs in the defaults/main.yaml?

Reply:
So, I've refactored this so that the user can set this in the job:

 - cifmw_update_ping_loss_second
 - cifmw_update_ping_loss_percent

Passing them as parameter to the scripts.